### PR TITLE
Documentation Bugs

### PR DESCRIPTION
**Pull request by led76marx**
Fixes Documentation Bugs #1645

### CODE REVIEW

**Review by Rajputusman — APPROVED** [review]
QA Passed.

File: src/containers/BankApi/BankApiAccounts/index.tsx [hunk 1]
```diff
@@ -12,8 +12,7 @@ const BankApiAccounts: FC = () => {
         generally pay lower transaction fees.
       </p>
       <p>
-        More information about account can be found in the <NavLink to="/guide/accounts">Accounts</NavLink> section of
-        the guide.
+        For more information, see <NavLink to="/guide/accounts">Accounts</NavLink>.
```
Comment (jamessspanggg) [inline src/containers/BankApi/BankApiAccounts/index.tsx:15]: can update the last updated date?

File: src/containers/BankApi/BankApiBlocks/index.tsx [hunk 2]
```diff
@@ -11,8 +11,7 @@ const BankApiBlocks: FC = () => {
         different recipients within a single block.
       </p>
       <p>
-        More information about blocks can be found in the <NavLink to="/guide/blocks">Blocks</NavLink> section of the
-        guide.
+        For more information, see <NavLink to="/guide/blocks">Blocks</NavLink>.
```
Comment (jamessspanggg) [inline src/containers/BankApi/BankApiBlocks/index.tsx:14]: same here, update last updated date

File: src/containers/BankApi/BankApiInvalidBlocks/index.tsx [hunk 3]
```diff
@@ -17,10 +16,6 @@ const BankApiInvalidBlocks: FC = () => {
         decrease the trust of their primary validator. The amount by which the primary validators trust is decreased is
         calculated based on the weighted trust of the sending confirmation validator.
       </p>
-      <p>
```
Comment (jamessspanggg) [inline src/containers/BankApi/BankApiInvalidBlocks/index.tsx:20]: same here

File: src/containers/BankApi/BankApiUpgradeNotice/index.tsx [hunk 4]
```diff
@@ -10,10 +9,6 @@ const BankApiUpgradeNotice: FC = () => {
         After a confirmation validator has upgraded to a primary validator, it will send out an upgrade notice to all
         connected banks.
       </p>
-      <p>
```
Comment (jamessspanggg) [inline src/containers/BankApi/BankApiUpgradeNotice/index.tsx:13]: here

File: src/containers/BankApi/BankApiValidators/index.tsx [hunk 5]
```diff
@@ -1,16 +1,11 @@
 import React, {FC} from 'react';
-import {NavLink} from 'react-router-dom';
 
 import {DocContainer, DocEndpoint, QueryParamsOffsetAndLimit, RequestResponseSnippet} from 'components';
 
 const BankApiValidators: FC = () => {
   return (
     <DocContainer className="BankApiValidators" title="Validators" lastUpdated="15 Jan 2021">
       <p>Banks will maintain a record of all validators, both primary and confirmation, that they are connected to.</p>
-      <p>
```
Comment (jamessspanggg) [inline src/containers/BankApi/BankApiValidators/index.tsx:10]: here

File: src/containers/ConfirmationValidatorApi/ConfirmationValidatorApiBankConfirmationServices/index.tsx [hunk 6]
```diff
@@ -15,10 +14,6 @@ const ConfirmationValidatorApiBankConfirmationServices: FC = () => {
         connected bank. Upon receiving this payment, it is implied that the sending bank is purchasing confirmation
         services.
       </p>
-      <p>
```
Comment (jamessspanggg) [inline src/containers/ConfirmationValidatorApi/ConfirmationValidatorApiBankConfirmationServices/index.tsx:18]: here

File: src/containers/ConfirmationValidatorApi/ConfirmationValidatorApiPrimaryValidatorUpdated/index.tsx [hunk 7]
```diff
@@ -14,10 +13,6 @@ const ConfirmationValidatorApiPrimaryValidatorUpdated: FC = () => {
         The last step in the resync process is the notice that is sent from banks to their CVs to indicate that they are
         leaving the current network and switching to a new PV (a new network).
       </p>
-      <p>
```
Comment (jamessspanggg) [inline src/containers/ConfirmationValidatorApi/ConfirmationValidatorApiPrimaryValidatorUpdated/index.tsx:17]: here

File: src/containers/ConfirmationValidatorApi/ConfirmationValidatorApiUpgradeRequest/index.tsx [hunk 8]
```diff
@@ -13,7 +11,6 @@ const ConfirmationValidatorApiUpgradeRequest: FC = () => {
         confirmation validators requesting one of them to upgrade to a primary validator. This is because the banks
         always prefer the most trusted validator to be the primary validator for the network.
       </p>
```
Comment (jamessspanggg) [inline src/containers/ConfirmationValidatorApi/ConfirmationValidatorApiUpgradeRequest/index.tsx:13]: here

**Comment by led76marx** [comment]
@jamessspanggg Hi - I made the updates per your request. However, my recommendation for small and tiny bug fixes like this one (which is a bunch of broken links) in tech docs is not to update the Last Updated dates. Reason being that audiences try to keep track of major updates per doc based on the last updated dates. There will be people that might try to keep track of a major update in these two APIs because of that. Please let me know if you agree and this makes sense to you, otherwise plz let me know and I will update dates for every update in docs no matter how big it is. Cheers!

**Comment by jamessspanggg** [comment]
> @jamessspanggg Hi - I made the updates per your request. However, my recommendation for small and tiny bug fixes like this one (which is a bunch of broken links) in tech docs is not to update the Last Updated dates. Reason being that audiences try to keep track of major updates per doc based on the last updated dates. There will be people that might try to identify a major update in these two APIs because of that. Please let me know if you agree and this makes sense to you, otherwise I will update dates for every update in docs no matter how big it is. Cheers!

Good point and I agree with you, actually I saw u changing some of the last updated date for small changes which I thought u forgot to do so for the others.

But yeah, agreed, tiny changes shouldn't change the last updated date.

**Review by jamessspanggg — APPROVED** [review]

**Comment by jamessspanggg** [comment]
hi @led76marx can u resolve the conflicts?

**Comment by led76marx** [comment]
@jamessspanggg fixed

**Review by jamessspanggg — APPROVED** [review]

**Pull request merged**